detectable/src/test/java/com/synopsys/integration/detectable/detectables/rubygems/gemlock/functional/GemlockNodeParserTest.java

```diff
@@ -38,31 +37,9 @@
 import com.synopsys.integration.detectable.annotations.FunctionalTest;
 import com.synopsys.integration.detectable.detectables.rubygems.gemlock.parse.GemlockParser;
 import com.synopsys.integration.detectable.util.FunctionalTestFiles;
-import com.synopsys.integration.detectable.util.GraphCompare;
 
 @FunctionalTest
 public class GemlockNodeParserTest {
```

Comment:
The other tests in this file seem to need the same treatment. They access large resource files with minimal assertions.